source/index.html.md

```diff
@@ -15,45 +16,37 @@ code_clipboard: true
 ---
 # Introduction
 
-Betable API is a RESTful service providing the ability to create a compliant wallet solution while powering CRM, fraud, and payment facilities for a regulated gaming solution.
+The Betable API is a RESTful service providing the ability to create a compliant gaming solution with integrated wallet, CRM, fraud, and payment facilities.
 
 ## Change History
 
 Version | Date | Summary of Changes
 --------- | ------- | -----------
-1.0 | 2020-03-23 | Initial release of Betable-LFS API.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+1.0 | 2020-03-23 | Initial release of Betable API.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
 
 ## Quick Reference
 
-All API access is over HTTPS, and accessed from the `ns-api.betable.com` domain. All API requests must be authenticated or a `401 Unauthorized error` will be returned (see Authentication).
+All API access is over HTTPS and must be authenticated or a `401 Unauthorized error` will be returned (see Authentication).
 
-A `User` corresponds to each unique player in the vendor system and contains profile and preference information.
+A `User` corresponds to each unique player in the partner system and contains profile and preference information.
 
 A `Session` is created for every login event by a player.
 
-A `Round` is a higher-level construct which can hold a series of wager and payout actions.
+A `Round` is a container for gaming interactions which typically consists of a series of wager and/or payout actions.
 
 A `Wager` is any entry or bet placed by a player.
 
 A `Payout` represents an amount won by a player.
 
 
-# API Overview
```

Comment:
I think `API Overview` should be kept as I like separating the Introduction from the Overview section (technical overhead vs just the book keeping).